src/frontend/src/pages/Root.tsx

```diff
@@ -0,0 +1,16 @@
+import Menu from "src/components/Menu";
+import { Outlet } from "react-router";
+
+const Home = () => {
+    return (
+        <div className="min-h-screen flex flex-col">
+            {/* TODO: get compareCount from app store */}
+            <Menu compareCount={0} />
+            <div className="flex flex-1 flex-col items-center self-stretch gap-6 rounded-t-[16px] bg-white px-10 py-16">
+                <Outlet />
```

Comment:
https://reactrouter.com/api/components/Outlet

Assuming the menu bar shows for all pages, this outlet is like a "slot" for rendering all children components that's nested under the root path, which is `/`- see the router definition in `App.tsx`